pkg/api/v2/types.go

```diff
@@ -156,13 +156,25 @@ type AlertmanagerSpec struct {
 type SeedSettings struct {
 	// the Seed level MLA (Monitoring, Logging, and Alerting) stack settings
 	MLA MLA `json:"mla"`
+	// the Seed level metering settings
+	Metering Metering `json:"metering"`
 }
 
 type MLA struct {
 	// whether the user cluster MLA (Monitoring, Logging & Alerting) stack is enabled in the seed
 	UserClusterMLAEnabled bool `json:"user_cluster_mla_enabled"`
 }
 
+type Metering struct {
```

Comment:
Why creating the new config, can't you reuse the same one from the metering package?

Reply:
Yes i agree